examples/se2_sam.cpp

```diff
@@ -451,18 +451,27 @@ int main()
          *  info matrix:
          *     W = I                    // trivial
          *
-         *  residual
-         *     r = W * (poses[0] (-) measurement) = poses[0].log()
+         *  residual uses left-minus since reference measurement is global
+         *     r = W * (poses[0] .- measurement) = log(poses[0] * Id.inv) = poses[0].log()
```

Comment:
@joansola How about `.(-)` ??

Reply:
Upper in this little doc section I define `.-` as left-minus and `-.` as right minus.

Reply:
I saw it, it is just that keeping the parentheses makes it clear at first sight that it is not a regular minus operator. Then it could be any variant `.(-)`, `(.-)`, `'(-)`, `('-)` etc.  
The operator `.-` reminds me of the element-wise operators in e.g. Matlab.

Reply:
OK let me take a look
There was a bug in the se3 file anyway. just pushed

Reply:
OK I pushed new minus signs as `(.-)` and `(-.)`.

Reply:
:+1: